wowup-electron/src/assets/i18n/en.json

```diff
@@ -133,11 +133,15 @@
         "TELEMETRY_DESCRIPTION": "Help improve WowUp by sending anonymous install data and/or errors.",
         "TELEMETRY_LABEL": "Telemetry",
         "TITLE": "Application",
-        "USE_HARDWARE_ACCELERATION_CONFIRMATION_LABEL": "Do you want to restart?",
-        "USE_HARDWARE_ACCELERATION_DESCRIPTION": "Disabling hardware acceleration might solve FPS issues and fix other rendering issues in this app. Changing this setting requires a restart.",
-        "USE_HARDWARE_ACCELERATION_DISABLE_CONFIRMATION_DESCRIPTION": "Disabling hardware acceleration requires the application to restart.",
-        "USE_HARDWARE_ACCELERATION_ENABLE_CONFIRMATION_DESCRIPTION": "Enabling hardware acceleration requires the application to restart.",
-        "USE_HARDWARE_ACCELERATION_LABEL": "Enable Hardware Acceleration"
+        "USE_HARDWARE_ACCELERATION_CONFIRMATION_LABEL" : "Do you want to restart?",
+        "USE_HARDWARE_ACCELERATION_ENABLE_CONFIRMATION_DESCRIPTION" : "Enabling hardware acceleration requires the application to restart.",
+        "USE_HARDWARE_ACCELERATION_DISABLE_CONFIRMATION_DESCRIPTION" : "Disabling hardware acceleration requires the application to restart.",
+        "USE_HARDWARE_ACCELERATION_LABEL" : "Enable Hardware Acceleration",
+        "USE_HARDWARE_ACCELERATION_DESCRIPTION" : "Disabling hardware acceleration might solve FPS issues and fix other rendering issues in this app. Changing this setting requires a restart.",
+        "START_WITH_SYSTEM_LABEL" : "Start WowUp with system",
+        "START_WITH_SYSTEM_DESCRIPTION" : "WowUp will start immediately after your operational ssytem is loaded",
+        "START_MINIMIZED_LABEL" : "Start WowUp minimized",
+        "START_MINIMIZED_DESCRIPTION" : "WowUp will start minimized on system startup"
```

Comment:
Labels and descriptions for new options

Reply:
You can sync translations by running `npm run i18n`, this will add the missing translations to all other available translations, and sort them